File: day01/jl/part1.jl
ret = read("input.txt",String)

#print(ret)

thefloor=0
for i in 1:length(ret)
	if ret[i]=='('
		global thefloor+=1
	elseif ret[i]==')'
		global thefloor-=1
	end
end

println("Final answer: $thefloor")

File: day01/py/part1.py
#Read data 
text_file = open("input.txt", "r")
lines = text_file.readlines()

mystring=lines[0]
mylist=[]
for i in range(len(mystring)):
	if mystring[i]=="(":
		mylist.append(1)
	elif mystring[i]==")":
		mylist.append(-1)

print("Final Answer:",sum(mylist))